Examples/AutomateTest/README.md

```diff
@@ -0,0 +1,12 @@
+The goal of this script is to automate the testing of react-native, react-native-code-push and their internal compatibility. Run **./run.sh** to create an app and test it. 
+
+You can configure below variables in the bash. 
```

Comment:
I don't think these make sense as environment variables - I don't think of them as things that need to be persisted across sessions, you really only want to set them once for the execution of the script. Instead, I recommend doing an input prompt that can be overriden by command-line arguments.

In other words:

```
prompt> ./generate-app.sh
What react-native version would you like to use (default: latest)? <Hit return key>
What react-native-code-push version would you like to use (default: latest)? 1.17.0-beta

Using:
react-native@0.41.0
react-native-code-push@1.17.0-beta
```

Can be overriden like this:

```
prompt> ./generate-app.sh latest 1.17.0-beta

Using:
react-native@0.41.0
react-native-code-push@1.17.0-beta
```